public/directives/minicolors.js

```diff
@@ -0,0 +1,114 @@
+// The MIT License (MIT)
```

Comment:
All third party code should exist under the `public/lib` folder. Please move this file under lib/jquery.minicolors